packages/test/test-end-to-end-tests/src/test/messageSize.spec.ts

```diff
@@ -253,6 +253,7 @@ describeCompat("Message size", "NoCompat", (getTestObjectProvider, apis) => {
 		},
 	);
 
+	// See ADO:8608
```

Comment:
Can you elaborate in PR description, what flakiness is caused by existing message size tests, and how does bringing that down make the tests more reliable?